web-config.yaml

```diff
@@ -184,8 +184,24 @@ stormpath:
       forgotUri: "/#/forgot"
       registerUri: "/#/register"
 
+
+    # Social provider configuration.  This defines the callback URIs for OAuth
+    # flows, and the scope that is requested of each provider.  These settings
+    # have no affect if the application does not have an account store for the
+    # given provider.
+
     socialProviders:
-      callbackRoot: "/callbacks"
+      facebook:
+        callbackUri: "/callbacks/facebook"
+        scope: "email"
+      github:
+        callbackUri: "/callbacks/github"
+        scope: "user:email"
+      google:
+        callbackUri: "/callbacks/google"
+        scope: "email profile"
```

Comment:
Some providers want space-separated, some want comma-separated.  This scope string value should be passed directly to the provider.  I'll update the comment on this section to make this clearer.